test/e2e/framework/pod/resource.go

```diff
@@ -35,6 +35,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	"k8s.io/kubernetes/pkg/client/conditions"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
```

Comment:
noting that long term we should stop importing kubernetes/pkg in the framework.

Reply:
Yes, we can try to bring up the error types into the CRI or stating repositories to enforce a uniform behavior around runtimes, which seems to diverge.